data/announcements/scheduled.yaml

```
@@ -34,9 +34,9 @@ announcements:
   startTime: 2022-04-07T00:00:00 # Added in PR: https://github.com/kubernetes/website/pull/32805
   endTime: 2022-05-10T00:00:00 # Expires 2022-05-10, but will change text slightly on release date
   style: "background: #326ce5"
-  title: "Dockershim removal set for Kubernetes 1.24"
+  title: "Dockershim removed from Kubernetes 1.24"
```

```suggestion
  title: "Dockershim removed from Kubernetes"
```
(To me) the following paragraph makes the context clear enough. We also link to a frequently-asked-questions page.

---

Fair enough. I've made the change you suggested.